types/rest/rest.go

```diff
@@ -111,13 +111,13 @@ func ReadRESTReq(w http.ResponseWriter, r *http.Request, cdc *codec.Codec, req i
 
 // ErrorResponse defines the attributes of a JSON error response.
 type ErrorResponse struct {
-	Code    int    `json:"code,omitempty"`
-	Message string `json:"message"`
+	Code  int    `json:"code,omitempty"`
+	Error string `json:"error"`
```

Comment:
ref: #3693 -- this should always be valid JSON @alessio 

/cc @faboweb

Reply:
yes plz!